src/tilesource.js

```diff
@@ -345,18 +345,17 @@ $.TileSource.prototype = {
      * @param {OpenSeadragon.Point} point
      */
     getTileAtPoint: function(level, point) {
+        var validPoint = point.x >= 0 && point.x <= 1 &&
+            point.y >= 0 && point.y <= 1 / this.aspectRatio;
+        $.console.assert(validPoint, "[TileSource.getTileAtPoint] called with invalid point " + point.toString());
```

Comment:
Assert messages should be about the requirement, not the failure, as in "I'm asserting that this point is valid". In this case, something like `must be called with valid point`.